ngff_zarr/cli.py

```diff
@@ -37,6 +37,7 @@ def main():
 
     metadata_group = parser.add_argument_group("metadata", "Specify output metadata")
     metadata_group.add_argument('-d', '--dims', nargs='+', help='Ordered OME-Zarr NGFF dimensions from {"t", "z", "y", "x", "c"}', metavar='DIM')
+    metadata_group.add_argument('-u', '--units', nargs='+', help='Ordered OME-Zarr NGFF axes spatial or temporal units', metavar='UNITS')
```

Comment:
To be more explicit on the comtmand line, can we have the same behavior as `--scale`, `--translation` here? i.e. `<dim1> <dim1value> <dim2> <dim2value> etc.`

Reply:
Good catch, updated.